apps/docs/src/pages/components/breadcrumb.mdx

```diff
@@ -0,0 +1,99 @@
+import {
+  breadCrumbClasses,
+  HvBreadCrumb,
+} from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("BreadCrumb", "core", breadCrumbClasses, [
+    "PathElement",
+  ]);
+  return { props: { ssg: { meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvBreadCrumb}
+  componentName="HvBreadCrumb"
+  controls={{
+    maxVisible: {
+      defaultValue: 4,
+    },
+  }}
+  componentProps={{
+    listRoute: [
+      { label: "Label 1", path: "route1" },
```

Comment:
can't we use the hash here? this way the user wouldn't be redirected await from the docs
```suggestion
      { label: "Label 1", path: "#route1" },
```